### ISSUE TITLE
Forum-Micmac is down

### ISSUE THREAD

**Issue by antochny** [micmacIGN/micmac#161]
Hello, 

I don't know to whom and how share this information to dev team (in charge of user's forum), but the forum hosted in [http://forum-micmac.forumprod.com/](url) is down, for few days at least weeks, maybe a month now.

Regard
Anthony

**Comment by kikislater** [comment]
It's on and off ... Not often online. Only european tome I guess. 
Weird access I recognize, I think it's selfhosted and work with wake on lan or when user who manage this forum decided to ne online ... Not a valuable ressource !

**Comment by antochny** [comment]
Well I guess I'm trying to visit only during off period for a month now ! Not reliable sure but still valuable ressource.
It' weird though because it used to be quite stable, this on-off period appeared in the past months.
I'll try to get in touch with Luc Girod to find more reliable host.

**Comment by kikislater** [comment]
Just a quick update. If you go to http://support.forumprod.com and http://support.forumprod.com/important-avenir-de-forumprod-t5554.html  you will see that these forums has been abandoned by his author ... And a lot of users are complaining about access before European timezone 7 to 9h ... SO may be not selfhosted by @luc-girod  but by owners of these forums. 

Btw domain name has been renewed (OVH) until 2023
```
$ whois forumprod.com | grep Expir
   Registry Expiry Date: 2023-07-08T23:52:40Z
Registrar Registration Expiration Date: 2023-07-08T21:52:40.0Z
```

**Comment by luc-girod** [comment]
Yes, forumprod is broken. That's why my forum signature and my automail when new people sign up includes a link to the subreddit and a stern recommendation to go there instead of the derelict forum.

**Comment by kikislater** [comment]
Ok but not all are kind fan of reddit (lot of tracking) ...

**Comment by luc-girod** [comment]
I get that, but Reddit does work, it is stable, efficient, searchable, easy to moderate and doesn't need maintenance from our part. And there's tracking everywhere, also on this very nice Github as it is Microsoft behind it anyways, and who knows what kind of security hell the forumprod is...

**Comment by antochny** [comment]
That's explain a lot ! Such a pitty, the forum host is derelict but a big part of its content is still useful...
1/ @luc-girod is there a way to migrate somehow the forum (I read that it could be complicated with Forumprod) or at least archive it in a "savage" way ? The MAP has been involved in this forum since the early beginning, we can probably make something to extend the sustainability (especially since our infrastructure is moving to a centralized data-center hosted at CNRS)
2/ like @kikislater I'm not huge fan of reddit. What about creating a new forum or similar system ? Once again the MAP can get involved. We still have the administrate and moderate but there is plenty of solution for free long term hosting nowadays (TGIR HumaNum etc).

 I do think that MicMac needs and deserves a user's support.

**Comment by kikislater** [comment]
> at least archive it

+1

> What about creating a new forum or similar system ? 

Thinking about it as well !

**Comment by luc-girod** [comment]
I had looked up how to save the content of the forum for later use, but it basically was impossible. The value of the info on there is why I haven't made effort to just close it down for now.
Maybe big boss @deseilligny has an opinion on the way forward?

**Comment by kikislater** [comment]
Otherwise may be web scrapping in that way !

**Comment by deseilligny** [comment]
To be honest I have no idea.  Maybe I can discuss it with IT expert like Christophe in CC.


________________________________
De : Luc Girod [notifications@github.com]
Envoyé : vendredi 5 mars 2021 09:54
À : micmacIGN/micmac
Cc : Marc Pierrot-Deseilligny; Mention
Objet : Re: [micmacIGN/micmac] Forum-Micmac is down (for a while) (#161)


I had looked up how to save the content of the forum for later use, but it basically was impossible. The value of the info on there is why I haven't made effort to just close it down for now.
Maybe big boss @deseilligny<https://github.com/deseilligny> has an opinion on the way forward?

**Comment by antochny** [comment]
@kikislater Yes web-scrapping was the savage method I was thinking too ;)
@luc-girod A kind of understood that ForumProd made no effort the help the migration since the end of support... I guess web-scrapping at the very last moment, just before switching to the "new forum" seems the only way.

According to MPD opinion, take as granted my personal involvement and the support of MAP-GAMSAU ;)

**Comment by luc-girod** [comment]
@antochny Works for me ;)

**Comment by meynardc** [comment]
Ok, scrapping is done.
I've an archive file (60MB .z or 100MB .zip) of all forum pages with internal links preserved.
Basically, you can locally browse the forum with any web browser.

Anyone interested ?

**Comment by DenisJouin** [comment]
Bonjour Meynardc,
Je suis intéressé par l'archive du forum. Comment y accéder ?
Merci pour le boulot
Denis Jouin

**Comment by kathanmehta007** [comment]
Hi @meynardc 

Thanks a lot for your help scraping the forum pages ! Those had a ton of relevant and important information and I am sure preserving all that is going to help a lot of people.
Would it be possible for me to have access of the zip file ?

Again, thanks a lot for your help !

**Comment by antochny** [comment]
Hi @meynardc,
When we will have the hand on the archive it would be relevant to make it accessible easily, for example, hosting it within the MicMac's Wiki ([https://micmac.ensg.eu/index.php/Accueil](https://micmac.ensg.eu/index.php/Accueil))

**Comment by meynardc** [comment]
Hi Kathan ! 

Sorry for my late answer, I was on vacation. 
The zip file (~100MB) contains the raw scrap result. I haven't done any filtering/cleaning. Many pages have duplicate names ( x.1, x.2, x.3 ...), probably because of download re-attempts. 
I can upload the zip file (~100MB) wherever you want. (I prefer not to make it publicly accessible under this form and I won't have time to improve that). 

Regards, 

-- 
Christophe MEYNARD - Chef du IGN/ENSG/CCIMI - UGE/LASTIG/ACTE 
Institut national de l'information géographique et forestière 
e-mail: ***@***.*** 


De: "Kathan Mehta" ***@***.***> 
À: "micmacIGN/micmac" ***@***.***> 
Cc: "Christophe Meynard" ***@***.***>, "Mention" ***@***.***> 
Envoyé: Mardi 1 Août 2023 15:01:29 
Objet: Re: [micmacIGN/micmac] Forum-Micmac is down (for a while) (#161) 





Hi [ https://github.com/meynardc | @meynardc ] 

Thanks a lot for your help scraping the forum pages ! Those had a ton of relevant and important information and I am sure preserving all that is going to help a lot of people. 
Would it be possible for me to have access of the zip file ? 

Again, thanks a lot for your help !

**Comment by llongour** [comment]
Perhaps it would be interesting to try an import on discourse (probably laborious, but I'm willing to take a look when I have the time). Especially since https://forum.geocommuns.fr/ is quite popular for this kind of project.

**Comment by antochny** [comment]
Very good idea, let us know if you have spare time to proceed the import !

**Comment by antochny** [comment]
Relauching this thread as I discovered that the forum is actually backed-up (maybe partially) in the Internet Archive Wayback Machine :
https://web.archive.org/web/20220000000000*/%20http://forum-micmac.forumprod.com/
